pkgs/top-level/all-packages.nix

```diff
@@ -41,18 +49,6 @@ with pkgs;
   # For convenience, allow callers to get the path to Nixpkgs.
   path = ../..;
 
-
-  ### Helper functions.
-  inherit lib config;
-
-  inherit (lib) lowPrio hiPrio appendToName makeOverridable;
```

Comment:
You cannot remove these from here as it could break out-of-tree code.

Reply:
Mhm. The problem is that every change in nixpkgs can break out-of-tree code.
The question is, how much breakage would be acceptable.